Docs for runner profiles (#2491)
hashicorp/waypoint: website/content/docs/runner/profiles.mdx

---
layout: docs
page_title: Runner Profiles
description: |-

> TODO(izaak) better: you can manage things with `waypoint runner profile`!

Reviewer: ```suggestion
Each runner profile entry represents the ability to spawn
runners (i.e. on-demand runners) when needed using the configured plugin.
```

---

# Runner profiles

With runner profiles, Waypoint's remote runners can be configured to spawn new ephemeral runner instances (i.e. on-demand runners)
to perform operations like building and deploying apps. Runner profiles describe the configuration and settings
necessary to create an on-demand runner. Profiles can be used globally, or by specific projects.

## Viewing runner profiles

> Waypoint installations performed on Kubernetes and ECS with the `waypoint install` command come with a default

Reviewer: I think this is true for the helm install too?

runner profile already configured. You can view runner profiles with the `waypoint runner profile list` command:

```shell-session
$ waypoint runner profile list

     NAME    | PLUGIN TYPE |           OCI URL             | DEFAULT
-------------+-------------+-------------------------------+----------
  kubernetes | kubernetes  | hashicorp/waypoint-odr:latest | yes
```

If a default runner profile is configured for a given plugin, Waypoint will use that profile to create a new on-demand

> runner for each operation that pertains to that plugin. For the majority of users, this default runner profile should

Reviewer: ```suggestion
runner for each operation on the platform for that plugin. For the majority of users, this default runner profile should
```
